src/main/java/com/arpnetworking/tsdcore/sinks/HttpSinkActor.java

```diff
@@ -160,6 +198,12 @@ public Receive createReceive() {
 
     private void processFailedRequest(final PostFailure failure) {
         _inflightRequestsCount--;
+
+        try (Metrics metrics = _metricsFactory.create()) {
+            metrics.incrementCounter(_samplesDroppedName, 1);
```

Comment:
Samples dropped is the number of measurements in the request. For aggregated data, this is represented either as:

1) The sum of the value of the count statistic (non-histogram data).
2) The sum of the bucket values (for histogram data).

You can see logic for this on the CAGG side where we compute the samples received:

https://github.com/ArpNetworking/metrics-cluster-aggregator/commit/e40d6dc9f88341a961b99b1953db6b07be55f301#diff-eba1035dfa98d0f4ef8e755ac5ce7f35R121